### PR TITLE
Rename of setGatewayConfigs

### PR DESCRIPTION
**Pull request by thenor**
Rename setGatewayConfigs to startGateways

### CODE REVIEW

**Review by kursatkobya — APPROVED** [review]

File: libsoftwarecontainer/include/softwarecontainer.h [hunk 1]
```diff
@@ -96,11 +96,12 @@ class SoftwareContainer :
     ObservableProperty<ContainerState> &getContainerState();
 
     /**
-     * @brief Set the gateway configurations and activates the configured gateway
+     * @brief Starts the Gateway by setting the gateway configurations 
+     * and activating the configured gateway
```
Comment (tobsan) [inline libsoftwarecontainer/include/softwarecontainer.h:100]: Should be plural here.

**Review by tobsan — APPROVED** [review]
Only one minor comment.

File: agent/src/softwarecontaineragent.h [hunk 2]
```diff
@@ -192,7 +192,7 @@ class SoftwareContainerAgent
      *
      * Capabilities are a mapping from a string to a set of gateway configurations. Capabilities
      * are translated into gateway configurations and then sent the same way as with
-     * setGatewayConfigs
+     * startGateways
```
Comment (thenor) [inline agent/src/softwarecontaineragent.h:195]: Not sure I understand why this sentence mentions the function name, but I just changed it to use the new function name.

File: libsoftwarecontainer/include/softwarecontainer.h (hunk 1 above)
Comment (thenor) [reply libsoftwarecontainer/include/softwarecontainer.h:100]: Good catch.

**Review by breakreturn — APPROVED** [review]

**Pull request merged**